docs/automation.md

```diff
@@ -0,0 +1,144 @@
+
+## Automation options
+
+These additional options are not mandatory but may be useful:
+
+|Option|Alias|Meaning|
+|:----|:----|:----|
+|`--schemaBaseUri`| |determines what base URI is used for the schema IDs being imported. If not provided, this will default to `https://demostore.amplience.com/`.|
+|`--omitExtensionEcommToolkit`|`-o`|Will automate your schemas without the eComm Toolkit extension so that you can manually type product references instead of browsing and selecting|
+
+## Automation result
+Once the automation has been ran, your account will be populated with content schemas, content types, dependant extensions, cards, icons and visualisations all pointing to defauly Stylitics accounts
+
+
+## Customisations & Implementation
+
+### Icons, Cards & Visualisations
+
+The automation comes with default icons, cards and visualisations. These are stand-alone examples to enable you to see get started quickly.
+
+When using for your implementation it is likely that you will want to ammend these to point to your application. You can ammend directly in each content-type in the Amplience Dynamic Content UI, or ammend in the automation.
+
+1. Browse to `amplience-automation/automation-files/type`
+2. Open up all of the `*.hbs` files
+3. Depending on what you would like to change, see the attributes [`icons`](#customising-icons), [`cards`](#customising-cards) or [`visualisations`](#customising-visualisations) object and ammend as appropriate.
+4. Save the files and run the automation
+
+#### Customising Icons
+
+![Stylitics Content Types)](../docs/media/stylitics-content-types.png)
+
+The files in `Assets` are all icons used by the content types. You can change to be any https url to an image of your choice. Feel free to use these as a basis.
+
+[Icons documentation](https://amplience.com/developers/docs/dev-tools/guides-tutorials/content-types/#choosing-an-icon)
+
+#### Customising Cards
+
+These are iframed HTML sources loaded into provide a preview of the content when selecting from the library view.
+
+[Cards documentation](https://amplience.com/developers/docs/dev-tools/guides-tutorials/content-types/#configuring-a-card)
+
+![Stylitics cards)](../docs/media/stylitics-cards.png)
+
+
+#### Customising Visualisations
+
+These are iframed HTML sources loaded when editing content to see the result of your content edits.
+
+[Visualisations documentation](https://amplience.com/developers/docs/dev-tools/guides-tutorials/content-types/#setting-up-visualizations)
+
+![Visualisations)](../docs/media/stylitics-crs-viz.png)
+
+### Account
+
+This automation comes with a dropdown and selector for some demo accounts from Stylitics to get you up and running fast and test a technical implementation and rendering in your application.
+
+If you are a Stylitics customer, will have different account(s) that you are using.
```

Comment:
This seems odd as a standalone paragraph.

Reply:
"If you are a Stylitics customer, the account dropdown will have different account(s) that you are using." maybe?

Reply:
"If you are a Stylitics customer, you'll want your own set of accounts to be choices on the content form." ?